src/ConnectorRuntime.ts

```diff
@@ -36,14 +36,7 @@ export class ConnectorRuntime extends Runtime<ConnectorRuntimeConfig> {
     private accountController: AccountController;
 
     private _transportServices: TransportServices;
-    public get transportServices(): TransportServices {
-        return this._transportServices;
-    }
-
     private _consumptionServices: ConsumptionServices;
```

Comment:
If you remove the getters I'd suggest giving the private members their proper names back, without the leading underscore.

Reply:
our naming sceme is _ for privates, nothing is proper about removing _ here.

Reply:
There are other private members named without any underscores.

Reply:
And this invalidates my point?